spyder/widgets/shortcutssummary.py

```diff
@@ -56,6 +56,144 @@ def __init__(self, parent=None):
             }"""
         self.setStyleSheet(style)
 
+
+        # Make the name of shortcut keys localizable
+        shortcut_names_map = {
```

Comment:
I don't think this is the right solution because we'd need to remember to update this dict every time a new shortcut is introduced anywhere in our source code.

Instead, we should generate this dict programmatically from our config options (present in `config/main.py`, see the `shortcuts` section) because every new shortcut needs to be registered there.

That way the shortcuts in `Preferences > Keyboard shortcuts` will be localized too.

Reply:
I have a more general approach for this as you suggest. I think it is better to close this PR as the solution needs to take into account the Spyder plugin API

Reply:
Agreed.